.github/workflows/benchmark.yml

```diff
@@ -0,0 +1,79 @@
+name: Python Benchmark
+
+on:
+  push:
+  pull_request:
+
+concurrency:
+  group: benchmark-${{ github.ref }}
+  cancel-in-progress: true
+
+defaults:
+  run:
+    working-directory: ./bindings/python
+    shell: bash
+
+jobs:
+  build:
+    # supercharge/mongodb-github-action requires containers so we don't test other platforms
```

Comment:
We can remove this comment and the strategy section since we're only running with one OS/Python version

Reply:
Done.